body/src/main/java/com/networknt/body/BodyHandler.java

```diff
@@ -106,6 +115,30 @@ public void handleRequest(final HttpServerExchange exchange) throws Exception {
                 setExchangeStatus(exchange, CONTENT_TYPE_MISMATCH, contentType);
                 return;
             }
+            // parse the body to formdata if content type is multipart/form-data or application/x-www-form-urlencoded
+        } else if (contentType != null &&
+                (contentType.startsWith("multipart/form-data") || contentType.startsWith("application/x-www-form-urlencoded"))) {
+            if (exchange.isInIoThread()) {
+                exchange.dispatch(this);
+                return;
+            }
+            exchange.startBlocking();
+            // Convert multipart/form-data to application/x-www-form-urlencoded to enable form parser
+            if (contentType.startsWith("multipart/form-data")) {
+                exchange.getRequestHeaders().put(Headers.CONTENT_TYPE, "application/x-www-form-urlencoded");
+            }
+            try {
+                FormParserFactory formParserFactory = FormParserFactory.builder().build();
+                FormDataParser parser = formParserFactory.createParser(exchange);
+                if (parser != null) {
+                    FormData data = parser.parseBlocking();
+                    exchange.putAttachment(REQUEST_BODY, data);
+                }
```

Comment:
Would it be possible to convert the FormData into a Map so the endpoint's receive a consistent data type?